docs/content/functioneel/zaken_api.md

```diff
@@ -0,0 +1,104 @@
+ZAKEN API
+===============================================
+**FUNCTIONELE DOCUMENTATIE**
+
+[aanzet tot documentatie van de API’s van het ZDS2-koppelvlak. Is vooralsnog een
+‘try-out’ n.a.v. [userstory
+\#188](https://github.com/VNG-Realisatie/gemma-zaken/issues/188) om inzicht te
+krijgen in de wijze van (functioneel) documenteren van API’s, als onderdeel van
+koppelvlakdocumentatie]
+
+| **Aspect**      | **Beschrijving**                                                                                                                                                                                  |
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Doel            | Het kunnen aanspreken van een voorziening voor het onderhouden en raadplegen van zaken inclusief bijbehorende statussen, rollen en relaties naar betrokkenen, objecten, zaaktypen en statustypen. |
+| Domein          | Zaakgericht werken                                                                                                                                                                                |
+| Provider        | [Zaakregistratiecomponent](https://www.gemmaonline.nl/index.php/GEMMA2/0.9/id-a97b6545-d5a7-485d-9b13-3ce22db5b9cf) (GEMMA2)                                                                      |
+| Consumer        | Componenten waarmee zaken behandeld worden (Zaakafhandelcomponenten; ZAC’s) en (andere) componenten die zaakgegevens raadplegen.                                                                  |
+| Informatiemodel | [RGBZ, versie 2.00.02](https://www.gemmaonline.nl/images/gemmaonline/a/a4/EgIM_20180621_-_ag_4_-_GEMMA_RGBZ_2.0_-_CONCEPT_20180613.zip) (in-ontwikkeling)                                         |
+| Specificaties   | <https://ref.tst.vng.cloud/zrc/api/v1/schema>                                                                                                                                                     |
+| Bijzonderheden  | Het RGBZ wordt in meerdere API’s uitgewerkt. De versie van het RGBZ waarnaar verwezen wordt omvat meer dan hetgeen in deze API is uitgewerkt.                                                     |
+
+Deze API omvat (vooralsnog) de resources Zaken, Rollen en Statussen.
+
+Elke resource biedt de mogelijkheid om objecten toe te voegen (POST), te wijzigen (PUT, PATCH), te verwijderen (DELETE) en te raadplegen (GET), tenzij anders vermeld.
```

Comment:
Wellicht is het niet nodig om POST, PUT, PATCH, DELETE en GET te noemen. Dit is een technische term (de HTTP-methode) om functioneel iets te bereiken, maar tegen ben ik ook niet als het tussen haakjes staat.

Reply:
Ik heb dit er in gezet om de link te leggen naar de technische documentatie. Dit zou dan ook de enige plek moeten zijn waar deze termen voor komen.

Reply:
Andere vermeldingen van deze operaties verwijderd, m.u.v. in Bijzonderheden van resource (DELETE, dat dat niet kan).